app/controllers/admin/content/events.js

```diff
@@ -0,0 +1,133 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import { later } from '@ember/runloop';
+
+export default Controller.extend({
+
+  subTopics: computed('model.topic', function() {
```

Comment:
Remove this computed property. Since the subTopics is loaded asynchronously, use an action that gets triggered when the topic dropdown is changed to make a call to the subtopics and set the subtopic once the promise is resolved.

Reply:
Ok, I will try doing this. :)